qemu/tests/throttle_multi_guests_parameter_test.py

```diff
@@ -0,0 +1,98 @@
+import os
+import logging
+import shutil
+
+from virttest import error_context
+from virttest import qemu_storage
+from virttest import data_dir
+from virttest import env_process
+
+from threading import Thread
+
+from provider.storage_benchmark import generate_instance
+from provider.throttle_utils import ThrottleGroupManager, ThrottleTester, \
+    ThrottleGroupsTester
+
+
+# This decorator makes the test function aware of context strings
+@error_context.context_aware
+def run(test, params, env):
+    """
+        Test throttle relevant properties feature.
+
+        1) Boot up guest with throttle groups.
+        There are two throttle groups and each have two disk
+        2) Build fio operation options and expected result
+         according to throttle properties.
+        3) Execute one disk or all disks testing on groups parallel.
+    """
+
+    def copy_base_vm_image():
```

Comment:
zixi, the vms preparation might be done with the following settings:
    start_vm = yes
    kill_vm = yes
    vms = vm1 vm2
    clone_master = yes
    master_images_clone = image1
    remove_image_image1 = yes
then vt can handle the vms, in test case, you just need to do some checkings/login, like:
   vms = env.get_all_vms()
    for vm in vms:
        vm.verify_alive()
        sessions.append(vm.wait_for_login())
then you can get all sessions

Reply:
remove_image_image1 = yes cannot remove system image when clone_master is set to yes, vt will only remove the clone images

Reply:
Thanks for your comment,  I tried clone_image=yes, but I found out I can't remove data images in the end due to codes in env_process.py.
if (not removed and params.get("remove_image", "yes") == "yes"):
        logging.info("Remove image on %s." % image.storage_type)
        if clone_master is None:
            image.remove()
        elif clone_master == "yes":
            if image_name in params.get("master_images_clone").split():
                image.remove()
Hence, remain the old way.